endpoints/users.go

```diff
@@ -0,0 +1,117 @@
+package endpoints
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"github.com/gin-gonic/gin"
+	"github.com/ariebrainware/paylist-api/model"
+	"golang.org/x/crypto/bcrypt"
+)
+
+//createUser
+func CreateUser(c *gin.Context) {
+	users := model.User{
+		Username: c.PostForm("username"),
+		Password: c.PostForm("password"),
+	}
+	fmt.Println(c.PostForm("username"))
+	fmt.Println(c.PostForm("password"))
+
+	db.Save(&users)
+	c.JSON(http.StatusCreated, gin.H{
+		"status":	http.StatusCreated,
+		"message":	"User created Successfully!",
+		"resourcedId": users.ID,
+	})
+}
+
+//GetUser
+func FetchUser(c *gin.Context) {
+	var users []model.User
+
+	db.Find(&users)
+
+	if len(users) <= 0 {
+		c.JSON(http.StatusNotFound, gin.H{"status": http.StatusNotFound, "message":"No user found!"})
+		return
+	}
+	c.JSON(http.StatusOK, gin.H{"status": http.StatusOK, "data": users})
+}
+
+//UpdateUser
+func UpdateUser(c *gin.Context) {
+	id, _ := strconv.Atoi(c.Param("id"))
+	updateuser := model.User{
+		Username: c.PostForm("username"),
```

Comment:
need more entity, rather than just update `username` and `password`. But i think `username` should not be updated because its kinda unique